javers-spring-jpa/src/main/java/org/javers/spring/jpa/JaversEntityListener.java

```diff
@@ -0,0 +1,130 @@
+package org.javers.spring.jpa;
+
+import org.javers.common.collections.Maps;
+import org.javers.core.Javers;
+import org.javers.spring.auditable.AuthorProvider;
+import org.javers.spring.auditable.CommitPropertiesProvider;
+import org.springframework.beans.factory.ObjectProvider;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Configurable;
+
+import javax.persistence.PostPersist;
+import javax.persistence.PostRemove;
+import javax.persistence.PostUpdate;
+import java.util.Map;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * JPA entity listener to commit auditing changes. To get this one flying be
+ * sure you add @EnableJaversEntityListeners to your application config
+ * and add this entity listener in your {@code orm.xml} as follows:
```

Comment:
again, xml is passé, is there another option to kick it of without xml?

Reply:
thats just a documentation how user can trigger this listener, if they like/need xml there is an option, which hibernate provides, not javers
and there are two options specified - xml or @EntityListeners, whichever user need
mb we should clarify this piece of javadoc if its confusing..